### ISSUE TITLE
Add a reminder/task

### ISSUE THREAD

**Issue by rahulsain** [rahulsain/SharedNotes#11]
Used it as a to-do list and sent a notification if user(alarm)

**Comment by Kajal13081** [comment]
Hey @rahulsain I want to work on this issue. Can you explain the issue a little bit more like how to approach the issue, the process?

**Comment by rahulsain** [comment]
Sure, I am currently thinking to adding reminder to this app, which shows notification on the specified time and user can put task on priority basis.

**Comment by Kajal13081** [comment]
> Sure, I am currently thinking to adding reminder to this app, which shows notification on the specified time and user can put task on priority basis.

Okay, so I would like to add this feature. Can you assign me for hactoberfest?

**Comment by rahulsain** [comment]
Sure, I will add the labels in your PR, but the PR should be valid , functional, and good code design

**Comment by rahulsain** [comment]
or you can work on this @cdhiraj40

**Comment by cdhiraj40** [comment]
> or you can work on this @cdhiraj40

i can sure work on remainders, is it possible to add remainder feature for now? and as i learn more functionality i can try to add more :)

**Comment by rahulsain** [comment]
> > or you can work on this @cdhiraj40
> 
> i can sure work on remainders, is it possible to add remainder feature for now? and as i learn more functionality i can try to add more :)

Yes, it is possible to add this reminder feature as it will be operated only offline

**Comment by cdhiraj40** [comment]
> > > or you can work on this @cdhiraj40
> > 
> > i can sure work on remainders, is it possible to add remainder feature for now? and as i learn more functionality i can try to add more :)
> 
> Yes, it is possible to add this reminder feature as it will be operated only offline as per now

do you want the remainder field in drop down menu it self or you have something on how it should be implemented

**Comment by rahulsain** [comment]
For the testing purpose you can do what suit you best, just do all your
work in new branch, I will help you along the way

**Comment by rahulsain** [comment]
I have checked the issue, you have to create your own project in firebase console, there you will get this json file
